examples/BertNewsClassification/README.md

```diff
@@ -87,15 +87,15 @@ create an empty directory `model_store` and run the following command to start t
 
 Run the following command to create a new deployment named `news_classification_test`
 
-`mlflow deployments create -t torchserve -m file:///home/ubuntu/mlflow-torchserve/examples/BertNewsClassification/models --name news_classification_test -C "MODEL_FILE=news_classifier.py" -C "HANDLER=news_classifier_handler.py"`
```

Comment:
Any specific reason for not using `file://` ?

Reply:
@shrinath-suresh, as discussed offline reverted this change